pkg/providers/snow/snow.go

```diff
@@ -79,6 +80,10 @@ func (p *snowProvider) SetupAndValidateCreateCluster(ctx context.Context, cluste
 		return fmt.Errorf("failed setting up credentials: %v", err)
 	}
 	p.setupMachineConfigs(clusterSpec)
```

Comment:
Do we still need this?

Reply:
yes

Reply:
yes, to set the managed-by annotation. but i can move it to cluster validation.